src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -281,18 +351,22 @@ public void close() {
             }
         }
 
+        for (ChannelHolder channelHolder: channelsByPin.values()) {
+            channelHolder.channel.abort();
+        }
```

Comment:
All channels should be closed automatically when we close the connection. Do we need to explicitly call `abort` method on the channel